custom_components/tuya_local/devices/minib_public_litterbox.yaml

```diff
@@ -0,0 +1,170 @@
+name: Litter box
+
+products:
+  - id: "93at0k76rfvteseb"
+    manufacturer: Tuya
```

Comment:
Leave the manufacturer out if you do not know. Tuya does not manufacture end products, just the WiFi modules.